near-sdk/src/collections/vector.rs

```diff
@@ -7,6 +7,7 @@ use std::marker::PhantomData;
 use borsh::{BorshDeserialize, BorshSerialize};
 
 use crate::collections::append_slice;
+use crate::store::IndexMap;
```

Comment:
This import is unused except with the `"unstable"` feature

```suggestion
#[cfg(feature = "unstable")]
use crate::store::IndexMap;
```

Reply:
CI should probably deny warnings, wdyt?